examples/cluster/plot_optics.py

```diff
@@ -0,0 +1,101 @@
+"""
+===================================
+Demo of OPTICS clustering algorithm
+===================================
+Finds core samples of high density and expands clusters from them.
+This example uses data that is generated so that the clusters have
+different densities.
+The clustering is first used in its automatic settings, which is the
+:class:`sklearn.cluster.OPTICS` algorithm, and then setting specific
```

Comment:
I don't think the term OPTICS algorithm is appropriate.

Reply:
reworded.